app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Admin::UsersController < Admin::ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->